index.html

```diff
@@ -8,7 +8,178 @@
     <title>PricingPage</title>
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap"
-      rel="stylesheet"/>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
+
+  <body>
+    <header>
+      <button class="btn-back">
+        <img class="img" src="img/Arrow Forward.svg" />Back
```

Comment:
img tags recommended to add alt attr, smt this case can warning in console, check it plz for all images